init_scripts/init-user.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+pip3 install django --user
```

Comment:
please learn about Python virtual environments and use them to manage your Python dependencies. I recommend using one of the modern Python dependency management tools such as `Pipenv` or `Poetry`, but you can use `requirements.txt`.